packages/host/app/components/pill.gts

```diff
@@ -5,85 +5,105 @@ import Component from '@glimmer/component';
 
 import {
   IconButton,
-  Pill,
+  Pill as BoxelPill,
```

Comment:
This looks like a code smell, let's not introduce another name for a Pill, but rather introduce new (sub)classes, which can be a CardPill and a FilePill.